dali/test/python/test_torch_pipeline_rnnt.py

```diff
@@ -331,7 +331,7 @@ def dali_reflect_pad_graph(x, pad_amount):
     return x
 
 
-@pipeline_def(batch_size=1, device_id=0, num_threads=3)
+@pipeline_def(batch_size=1, device_id=0, num_threads=4, exec_dynamic=True)
```

Comment:
exec_dynamic like 4 threads more :)

Reply:
Is it inefficient when 3 or less? How there `4` threads map to `3` in the old executor?